contributing/code/standards.rst

```diff
@@ -100,6 +100,9 @@ Structure
 
 * Place unary operators (``!``, ``--``, ...) adjacent to the affected variable;
 
+* Use `Yoda condition`_ when checking a variable against an expression to avoid
```

Comment:
I guess the correct wording would be "Use Yoda conditions" or "Use a Yoda condition" (the first sounds more natural to me).